### ISSUE TITLE
Difference in evalution

### ISSUE THREAD

**Issue by VincentItaliano** [PJLallen/D2GPLand#1]
Hello, I evaluated the model with your checkpoints on [Google Drive](https://drive.google.com/drive/folders/1Mll-izyMLoCnTxfW5LOJhzaThipnUSg0?usp=drive_link) and I found different metric values from those in the paper 
`mIOU = 51.4 (instead of 48.68) and mDice = 65.8 (instead of 59.38)`

Do you have any explanation ? Maybe you have uploaded a different checkpoint (better than the one in the paper) from the one you mentioned in the paper

**Comment by cuiruize** [comment]
Later, we have conducted parameter tuning and re-trained our model to improve the performance of the landmark detection. We have already reverted the Git version to apply all the hyperparameters that we used in our MICCAI work. You may download the code and try the evaluation again. Thanks for your  attention.
